### ISSUE TITLE
Color contrast does not meet minimum requirement - disabled previous paging links at bottom of search results

### ISSUE THREAD

**Issue by elynema** [avalonmediasystem/avalon#5573]
# Description

WCAG 2.1 level: AA

Pages encountered: search / browse results page

Color contrast does not meet minimum requirement
There is insufficient color contrast between the text and background on this page to meet the minimum contrast requirement.

Found with SiteImprove browser plugin Accessibility Checker. One the first page of results, the Previous link and the page 1 link at the bottom of the search results are disabled, gray text on gray does not provide enough color contrast. Same issue occurs for the Next link and the page 2 link when you are on the last page of results.

<img width="1229" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNnpCQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--bb42a113687bae4c01bbfc1c4fcd9eb7def7b07e/image.png" alt="image.png" />

<img width="1229" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNnJCQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--d27db7c6443607acff452062329fe291126944fc/image.png" alt="image.png" />

**Comment by joncameron** [comment]
aria-disabled should be added as well to this (just having the element be disabled is not enough). May be Blacklight overrides.

**Comment by joncameron** [comment]
<img width="804" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMXZ4QWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--612c71aea0da3a15ecd2aef064360be67abd8bfe/Screenshot%202024-01-23%20at%204.10.01%E2%80%AFPM.png" alt="Screenshot 2024-01-23 at 4.10.01 PM.png" />
The text color is still gray on avalon-dev for the Previous and Next buttons when they're disabled. 

@cjcolvar can you take a look?

**Comment by elynema** [comment]
@joncameron Chris disabled these, rather than changing the text color. SiteImprove is no longer complaining about color for this disabled element.
